docs/reference/modules/asset.md

```diff
@@ -0,0 +1,205 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/asset`
+
+**Alias:** `apos.asset`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+The `asset` module serves to organize, process, and output all project script and styling assets during the build process. In addition, it provides access to modify the project webpack configuration and exposes two CLI tasks for project building and webpack cache clearing. Options are passed through the creation of a `modules/@apostrophecms/asset/index.js` file.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`refreshOnRestart`](#refreshonrestart) | Boolean | If set to `true`, the browser will refresh on Apostrophe app restart. |
+| [`watch`](#watch) | Boolean | If set to `false`, none of the UI assets will be watched for triggering a restart. |
+| `watchDebounceMs` | Integer | Time in milliseconds to wait before re-triggering a restart on asset change. |
+
+### `refreshOnRestart`
+
+If `refreshOnRestart` is truthy and `process.env.NODE_ENV` is not set to `production`, then restart of the Apostrophe app will trigger a browser refresh.
```

Comment:
"This is useful in combination with `nodemon.`" Without this note it's not very helpful to document.

Reply:
Always document what the default behavior is.

Reply:
Changed. Not sure if you would like to see further comments here.